### PR TITLE
1193 Assorted Bug Fixes / PDF Styles

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2020-01-10 17 05 34
Screenshot 2020-01-10 17 05 44

### CODE REVIEW

File: shared/src/business/utilities/noticeOfTrialIssued.pug [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.clear
```
Comment (matthopson) [inline shared/src/business/utilities/noticeOfTrialIssued.pug]: Why's this duplicated?

**Review by matthopson — APPROVED** [review]
One question about a file that appears to duplicate some of the notice template. Not sure what that's for.

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/utilities/noticeOfTrialIssued.pug (hunk 1 above)
Comment (kkoskelin) [reply shared/src/business/utilities/noticeOfTrialIssued.pug]: I think I just need more coffee but can you talk about this some more, Matt?  I don't understand.

Comment (wbyoko) [reply shared/src/business/utilities/noticeOfTrialIssued.pug]: Do you mean the 'pdf header' or the '.clear'?

Comment (matthopson) [reply shared/src/business/utilities/noticeOfTrialIssued.pug]: This entire template looks like a duplication of lines 26-64 of `noticeOfTrialIssued.pug` above.

**Review by matthopson — APPROVED** [review]

File: shared/src/business/utilities/noticeOfTrialIssued.pug (hunk 1 above)
Comment (wbyoko) [reply shared/src/business/utilities/noticeOfTrialIssued.pug]: Yup, this file can be removed. Good catch!

**Pull request merged**